src/test/regress/multi_mx_schedule

```diff
@@ -36,7 +36,7 @@ test: recursive_dml_queries_mx multi_mx_truncate_from_worker
 test: multi_mx_repartition_udt_prepare mx_foreign_key_to_reference_table
 test: multi_mx_repartition_join_w1 multi_mx_repartition_join_w2 multi_mx_repartition_udt_w1 multi_mx_repartition_udt_w2
 test: multi_mx_metadata
-test: multi_mx_call
+test: multi_mx_call master_evaluation
```

Comment:
Worth adding a comment somewhere that this test relies on metadata being synced.

Reply:
I added on top of `master_evaluation .sql`